docs/admin/templates/extending-templates/parameters.md

```diff
@@ -313,6 +313,58 @@ data "coder_parameter" "project_id" {
 }
 ```
 
+## Workspace presets
+
+Workspace presets allow you to select settings tailored to your most common development scenarios.
+
+Coder admins can configure workspace presets with combinations of parameter values and workspace settings within each template's Terraform file, and workspace users can customize their configurations as they need to.
+
+Use `coder_workspace_preset` to define the preset parameters.
+
+<details><summary>Expand for an example</summary>
+
+```tf
+data "coder_workspace_preset" "goland-gpu" {
+  name        = "GoLand with GPU"
+  parameters = {
+    "machine_type"  = "n1-standard-1"
+    "attach_gpu"    = "true"
+    "gcp_region"    = "europe-west4-c"
+    "jetbrains_ide" = "GO"
+  }
+}
+
+data "coder_parameter" "machine_type" {
+  name          = "machine_type"
+  display_name  = "Machine Type"
+  type          = "string"
+  default       = "n1-standard-2"
+}
+
+data "coder_workspace_preset" "attach_gpu" {
+  name          = "attach_gpu"
+  display_name  = "Attach GPU?"
+  type          = "bool"
+  default       = "false"
+}
+
+data "coder_workspace_preset" "gcp_region" {
+  name          = "gcp_region"
+  display_name  = "Machine Type"
+  type          = "string"
+  default       = "n1-standard-2"
+}
+
+data "coder_workspace_preset" "jetbrains_ide" {
+  name          = "jetbrains_ide"
+  display_name  = "Machine Type"
+  type          = "string"
+  default       = "n1-standard-2"
+}
```

Comment:
These look like mislabeled `coder_parameter`s; this syntax will cause a build failure.